src/modules/PulseTransfer/PulseTransferModule.cpp

```diff
@@ -178,12 +178,31 @@ void PulseTransferModule::run(unsigned int event_num) {
             auto pulse_graph = new TGraph(static_cast<int>(pulse_vec.size()), &time[0], &pulse_vec[0]);
             pulse_graph->GetXaxis()->SetTitle("t [ns]");
             pulse_graph->GetYaxis()->SetTitle("Q_{ind} [e]");
-            pulse_graph->SetTitle(("Induced charge in pixel (" + std::to_string(index.x()) + "," +
+            pulse_graph->SetTitle(("Induced charge carriers per unit step time in pixel (" + std::to_string(index.x()) + "," +
                                    std::to_string(index.y()) +
-                                   "), Q_{tot} = " + std::to_string(pixel_index_pulse.second.getCharge()) + " e")
+                                   "), Q_{tot} = " + std::to_string(pixel_index_pulse.second.getCharge()) + " e (" + std::to_string(pixel_index_pulse.second.getCharge() * (-1.602e-4)) + " fC)")
```

Comment:
Instead of doing the conversion and printing of units manually you can simply do:

```cpp
Units::display(pixel_index_pulse.second.getCharge(), {"fC"})
```

which will convert from internal units to whatever unit you give it and will return a string with the units appended already